src/test/java/com/garrytrue/TestUtils.java

```diff
@@ -0,0 +1,17 @@
+package com.garrytrue;
+
+import com.garrytrue.model.Student;
+
+/**
+ * Created by garrytrue on 12.03.17.
+ */
+public final class TestUtils {
```

Comment:
This class must provide you generation of data not only inmemory but also for repository persistance